rest-api-spec/src/main/resources/rest-api-spec/api/_global.json

```diff
@@ -0,0 +1,29 @@
+{
+  "description": "Parameters that are accepted by all API endpoints.",
+  "documentation": "https://www.elastic.co/guide/en/elasticsearch/reference/current/common-options.html",
+  "params": {
+    "pretty": {
+      "type": "boolean",
+      "description": "Pretty format the returned JSON response.",
+      "default": false
+    },
+    "human": {
+      "type": "boolean",
+      "description": "Return human readable values for statistics.",
+      "default": true
+    },
+    "error_trace": {
+      "type": "boolean",
+      "description": "Include the stack trace of returned errors.",
+      "default": false
+    },
+    "source": {
+      "type": "string",
+      "description": "The URL-encoded request definition. Useful for libraries that do not accept a request body for non-POST requests."
+    },
+    "filter_path": {
```

Comment:
We support `ignore` as a concept just not through a url parameter. When we did have yaml rest test these were translated to a request config parameter. Agreed `ignore` should not be part of this list and maybe add it as a documentation note to the rest spec that any client implementation needs to have a mechanism to ignore bad status codes.